astra/src/main/java/com/slack/astra/bulkIngestApi/BulkIngestKafkaProducer.java

```diff
@@ -331,7 +179,8 @@ private BulkIngestResponse produceDocuments(
       // we will limit producing documents 1 thread at a time
       for (Trace.Span doc : indexDoc.getValue()) {
         ProducerRecord<String, byte[]> producerRecord =
-            new ProducerRecord<>(kafkaConfig.getKafkaTopic(), partition, index, doc.toByteArray());
+            new ProducerRecord<>(
+                super.kafkaConfig.getKafkaTopic(), partition, index, doc.toByteArray());
```

Comment:
I think the super here isn't necessary
```suggestion
               kafkaConfig.getKafkaTopic(), partition, index, doc.toByteArray());
```